src/test/java/org/opensearch/flowframework/FlowFrameworkRestTestCase.java

```diff
@@ -345,16 +345,24 @@ protected Response createWorkflow(RestClient client, Template template) throws E
      * Helper method to invoke the Create Workflow Rest Action without validation
      * @param client the rest client
      * @param useCase the usecase to create
+     * @param the required params
      * @throws Exception if the request fails
      * @return a rest response
      */
-    protected Response createWorkflowWithUseCase(RestClient client, String useCase) throws Exception {
+    protected Response createWorkflowWithUseCase(RestClient client, String useCase, List<String> params) throws Exception {
+
+        StringBuilder sb = new StringBuilder();
+        for (String param : params) {
+            sb.append("\"" + param + "\" : \"\"").append(",");
```

Comment:
Mixing StringBuilder append and string concatenation seems wrong somehow.   And appending characters is cleaner than appending escaped strings.   Can we do `.append('"').append(param)` and so on?